docs/Makefile

```diff
@@ -1,7 +1,7 @@
-DOCKER       = docker
-HUGO_VERSION = 0.50
-DOCKER_IMAGE = jojomi/hugo:$(HUGO_VERSION)
-DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(PWD):/src
+DOCKER       ?= docker
+HUGO_VERSION ?= latest
```

Comment:
Two issues with using `latest`:

1.  Things may break with a new version of Hugo
2. `latest` tag may be different for different users, unless everyone always does a `docker pull`